### PR TITLE
WIP - experimentation with Java 11

### PR DESCRIPTION
**Pull request by jgallimore**

### CODE REVIEW

File: examples/mp-rest-client/pom.xml [hunk 1]
```diff
@@ -52,8 +52,18 @@
             <version>${microprofile.rest-client.version}</version>
             <scope>provided</scope>
         </dependency>
-
-
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.3.0</version>
```
Comment (otaviojava) [inline examples/mp-rest-client/pom.xml]: maybe in the future, we need to join these version on just one place

File: container/openejb-core/pom.xml [hunk 2]
```diff
@@ -429,6 +429,34 @@
   </build>
 
   <dependencies>
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
```
Comment (jgallimore) [inline container/openejb-core/pom.xml]:
```
<dependency>
      <groupId>jakarta.xml.bind</groupId>
      <artifactId>jakarta.xml.bind-api</artifactId>
      <scope>provided</scope>
      <exclusions>
        <exclusion> <!-- Is already included in java-ee-api -->
          <groupId>jakarta.activation</groupId>
          <artifactId>jakarta.activation-api</artifactId>
        </exclusion>
      </exclusions>
    </dependency>
```

File: container/openejb-core/pom.xml [hunk 3]
```diff
@@ -429,6 +429,34 @@
...
+      <artifactId>jaxb-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish.jaxb</groupId>
+      <artifactId>jaxb-runtime</artifactId>
+    </dependency>
+    <dependency>
```
Comment (jgallimore) [inline container/openejb-core/pom.xml]: activation-api should be excluded?

File: container/openejb-core/pom.xml [hunk 4]
```diff
@@ -429,6 +429,34 @@
...
+      <groupId>javax.activation</groupId>
+      <artifactId>javax.activation-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.sun.activation</groupId>
+      <artifactId>javax.activation</artifactId>
+    </dependency>
+    <dependency>
```
Comment (jgallimore) [inline container/openejb-core/pom.xml]: Is com.sun.xml.ws required with the new CXF?

File: container/openejb-core/pom.xml [hunk 5]
```diff
@@ -429,6 +429,34 @@
...
+      <groupId>com.sun.xml.ws</groupId>
+      <artifactId>rt</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jacorb</groupId>
+      <artifactId>jacorb</artifactId>
+    </dependency>
+    <dependency>
```
Comment (jgallimore) [inline container/openejb-core/pom.xml]: Check and see if the RMI libraries are actually needed/used.

File: container/openejb-jee-accessors/pom.xml [hunk 6]
```diff
@@ -56,6 +56,16 @@
       <artifactId>javaee-api</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
```
Comment (jgallimore) [inline container/openejb-jee-accessors/pom.xml]:
```
<dependency>
      <groupId>jakarta.xml.bind</groupId>
      <artifactId>jakarta.xml.bind-api</artifactId>
      <scope>provided</scope>
      <exclusions>
        <exclusion> <!-- Is already included in java-ee-api -->
          <groupId>jakarta.activation</groupId>
          <artifactId>jakarta.activation-api</artifactId>
        </exclusion>
      </exclusions>
    </dependency>
```

File: examples/moviefun-rest/pom.xml [hunk 7]
```diff
@@ -171,8 +171,20 @@
...
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+      <version>2.3.0</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish.jaxb</groupId>
+      <artifactId>jaxb-runtime</artifactId>
+      <version>2.4.0-b180830.0438</version>
```
Comment (jgallimore) [inline examples/moviefun-rest/pom.xml]: Check version number, I think this gives a conflict

File: examples/moviefun-rest/pom.xml [hunk 8]
```diff
@@ -171,8 +171,20 @@
...
     <dependency>
       <groupId>org.apache.tomee</groupId>
-            <artifactId>javaee-api</artifactId>
-       <version>8.0</version>
+      <artifactId>javaee-api</artifactId>
+      <version>8.0</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
```
Comment (jgallimore) [inline examples/moviefun-rest/pom.xml]:
```
<dependency>
      <groupId>jakarta.xml.bind</groupId>
      <artifactId>jakarta.xml.bind-api</artifactId>
      <scope>provided</scope>
      <exclusions>
        <exclusion> <!-- Is already included in java-ee-api -->
          <groupId>jakarta.activation</groupId>
          <artifactId>jakarta.activation-api</artifactId>
        </exclusion>
      </exclusions>
    </dependency>
```

File: examples/mp-rest-client/pom.xml [hunk 9]
```diff
@@ -52,8 +52,18 @@
             <version>${microprofile.rest-client.version}</version>
             <scope>provided</scope>
         </dependency>
-
-
+        <dependency>
```
Comment (jgallimore) [inline examples/mp-rest-client/pom.xml]:
```
<dependency>
      <groupId>jakarta.xml.bind</groupId>
      <artifactId>jakarta.xml.bind-api</artifactId>
      <scope>provided</scope>
      <exclusions>
        <exclusion> <!-- Is already included in java-ee-api -->
          <groupId>jakarta.activation</groupId>
          <artifactId>jakarta.activation-api</artifactId>
        </exclusion>
      </exclusions>
    </dependency>
```

File: examples/moviefun-rest/pom.xml (hunk 7 above)
Comment (j4fm) [reply examples/moviefun-rest/pom.xml]: Should be 2.3.2 for now

**Comment by jgallimore** [comment]
> (I think) PR 419 already does this as well as other related needed dependency changes.

Yes, I'm wondering if this now redundant. Running tests to confirm.

File: container/openejb-core/pom.xml (hunk 5 above)
Comment (jgallimore) [reply container/openejb-core/pom.xml]: They're needed.

**Comment by AndyGee** [comment]
@jgallimore 12 is out now ;-)

**Comment by jeanouii** [comment]
Closing this one and I'll open a new one from the apache branch

**Pull request closed without merging**